sample/Kmd.Logic.Cpr.Events.Receiver/Kmd.Logic.Cpr.Events.Receiver.csproj

```diff
@@ -1,13 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
-    <UserSecretsId>af370594-5b69-44aa-8fcd-befcdff9cf8b</UserSecretsId>
```

Comment:
We don't need UserSecrets here?

Reply:
No. The application doesn't have any values that it takes from the appsettings except for the optional TenantName which is not a secret